File: tests/tests_old_compiler/RemoteCalls/J2C_NoReturnResults/jt5.c
char *compyou(char *s);


jtask* localyou(int c, char *s) {
    while(1) {
        jsleep(10000000);
        printf("############-->>> Hello YOU  %d, %s\n", c, s);
    }
}

jtask* testfunc(int x) {
    printf("------------ Value of x %d\n", x);
}


int main(int argc, char *argv[])
{
    localyou(10, "local-message...");
    return 0;
}

File: tests/Integration/temp/jt5.js
let count = 10;

jtask function compyou(str) {
    count++;
    console.log(count, str);
    return "hello -- ";
}

setInterval(()=> {
    console.log("hello.. main loop.. calling the C function testfunc..  ");
    let x = get_a_value(count++);
    x.then((y)=> {
        console.log("Sdsdasdasdasdsd ", y);
    })
    console.log("Return value ", x);
}, 100);
